src/main.js

```diff
@@ -4,5 +4,8 @@ export { default as ActionItem } from './components/ActionItem';
 export { default as NavigationButton } from './components/NavigationButton';
 export { default as Label } from './components/Label';
 
+// Layouts
+export { default as AbsoluteLayout } from './layouts/AbsoluteLayout';
```

Comment:
No need to import/export components from main.js
we should import and use them directly in the components when we need